File: app/src/main/java/com/example/a18/path/scratchview/ScratchActivity.java
package com.example.a18.path.scratchview;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.a18.path.R;

public class ScratchActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_scratch);

        ScratchView viewById = (ScratchView) findViewById(R.id.sv);
        viewById.setImageResource(R.drawable.k);
    }

}

File: m_scratch/src/main/java/pdog18/com/module_scratch/ScratchActivity.kt
package pdog18.com.module_scratch

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class ScratchActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_scratch)

        val viewById = findViewById<ScratchView>(R.id.sv)
        viewById.setImageResource(R.drawable.k)
    }
}
